### DISCUSSION TITLE
Import csv (project) - metafields stay empty

### DISCUSSION THREAD

**Discussion by markus7777777** [kimai/kimai#3656] [Questions & Answers]
Hello!

I imported a .csv file into kimai and  noticed that the metafields remain empty although the csv contains the metafields.
The default columns like name, customer and description get imported. However, the metafields (custom fields) **remain empty.**

### used command:
**bin/console kimai:import:project xxx.csv --reader=csv-semicolon**

### Column names of csv:
**name;description;orderdate;customer;**meta**.verantwortlich;**meta**.gemeinde;**meta**.kg_name;**meta**.auftragsart**

All internal meta-fields are lowercase and there is no leading blank/space between the header names (as seen above).

Custom fields Version: 1.24.2
Kimai Version: 1.29.0

Has anybody experience with this? 
Thanks in advance!

**Comment by kevinpapst** [comment]
share an example csv please

**Reply by markus7777777** [reply]
Here is sample.csv:

_name;description;orderdate;customer;meta.verantwortlich;meta.gemeinde;meta.kg_name;meta.auftragsart
9999;KG Muster 352/3, 356/2, 355/2;2010-11-30;Test Klient und Klientin;KRA;1234, Mustergemeinde;1234, MusterKG;AuftragsartA
10654;MusterKG, Test Straße, 57, Gst: 9999;1909-06-01;Kein Eintrag; Rotter;;;_

which leads to:

<img width="628" alt="image" src="https://user-images.githubusercontent.com/46875900/205288085-f136d160-8ae7-47fd-8491-ee84df84c852.png">

and:

<img width="709" alt="image" src="https://user-images.githubusercontent.com/46875900/205288332-193cda61-b4b2-444b-83a5-f91c174293dd.png">

**Comment by kevinpapst** [comment]
This discussion is outdated
